### PR TITLE
Update webpack config

### PR DESCRIPTION
**Pull request by rodfersou**

### CODE REVIEW

**Comment by rodfersou** [comment]
I don't know why this test is failing, it just can't find the registered viewlet

**Comment by rodfersou** [comment]
locally it works fine

**Comment by rodfersou** [comment]
looks like this is the error we are having here:
```
ERROR in ./app/brasilgovtiles.scss
Module build failed: ModuleBuildError: Module build failed: Error: Command failed: /home/travis/build/plonegovbr/brasil.gov.tiles/webpack/node_modules/mozjpeg/vendor/cjpeg
/home/travis/build/plonegovbr/brasil.gov.tiles/webpack/node_modules/mozjpeg/vendor/cjpeg: error while loading shared libraries: libpng16.so.16: cannot open shared object file: No such file or directory
```

File: webpack/app/scss/tiles/_videogallery.scss [hunk 1]
```diff
@@ -78,7 +78,7 @@
       padding-right: 25px;
 
       color: #D1E2FF !important;
-      background: url(./img/footer-arrow.jpg) no-repeat scroll right 0 transparent;
+      background: url(./img/footer-arrow_jpg.png) no-repeat scroll right 0 transparent;
```
Comment (rodfersou) [inline webpack/app/scss/tiles/_videogallery.scss]: there are another different image wwith name ``footer-arrow.jpg``

**Comment by rodfersou** [comment]
Need some help here, there is still an error that can't find the viewlet from travis:
```python
Running collective.cover.testing.collective.cover:Integration tests:
  Tear down brasil.gov.tiles.testing.brasil.gov.tiles:Robot in 0.000 seconds.
  Tear down plone.app.robotframework.remote.AutoLoginRemoteLibrary:RobotRemote in 0.005 seconds.
  Tear down plone.testing.z2.ZServer in 5.075 seconds.
  Tear down brasil.gov.tiles.testing.Fixture in 0.007 seconds.
  Set up collective.cover.testing.Fixture in 6.445 seconds.
  Set up collective.cover.testing.collective.cover:Integration in 0.000 seconds.


Error in test test_viewlet (brasil.gov.tiles.tests.test_resources_viewlet.ResourcesViewletTestCase)
Traceback (most recent call last):
  File "/opt/python/2.7.14/lib/python2.7/unittest/case.py", line 320, in run
    self.setUp()
  File "/home/travis/build/plonegovbr/brasil.gov.tiles/src/brasil/gov/tiles/tests/test_resources_viewlet.py", line 24, in setUp
    self.viewlet = self.get_viewlet(self.cover)
  File "/home/travis/build/plonegovbr/brasil.gov.tiles/src/brasil/gov/tiles/tests/test_resources_viewlet.py", line 37, in get_viewlet
    assert len(viewlet) == 1  # nosec
AssertionError

  Ran 1 tests with 0 failures and 1 errors in 0.060 seconds.
Tearing down left over layers:
  Tear down collective.cover.testing.collective.cover:Integration in 0.000 seconds.
  Tear down collective.cover.testing.Fixture in 0.007 seconds.
  Tear down plone.app.testing.layers.PloneFixture in 0.069 seconds.
  Tear down plone.testing.z2.Startup in 0.006 seconds.
  Tear down plone.testing.zca.LayerCleanup in 0.003 seconds.
Total: 90 tests, 0 failures, 1 errors in 3 minutes 47.584 seconds.
```

**Comment by rodfersou** [comment]
just checked again, and this error happen locally too

**Comment by rodfersou** [comment]
finally green!

File: src/brasil/gov/tiles/tests/test_upgrades.py [hunk 2]
```diff
@@ -64,11 +64,9 @@ def test_update_resources_references(self):
         _rename_resources(js_tool, RESOURCES_TO_UPDATE_INVERSE)
 
         css_ids = css_tool.getResourceIds()
-        self.assertIn('++resource++brasil.gov.tiles/tiles.css', css_ids)
```
Comment (hvelarde) [inline src/brasil/gov/tiles/tests/test_upgrades.py]: you should not mess around old upgrade steps.

**Pull request merged**